src/main/java/com/github/fred84/queue/logging/LogContextHelper.java

```diff
@@ -0,0 +1,10 @@
+package com.github.fred84.queue.logging;
+
+import java.util.Map;
+
+public interface LogContextHelper {
+
+    Map<String, String> getLogContext();
```

Comment:
getLogContext => get
we can't set anything except log context content itself, so why should we use name both in method and container

Reply:
setLogContext => set